operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -576,11 +562,22 @@ private void buildIngressController(String name, String domain,
         createOrEdit(builder.build(), existing);
     }
 
+    // for testing
+    void setAzReplicaCount(Optional<Integer> value) {
+        azReplicaCount = value;
+    }
+
     int numReplicasForZone(LongSummaryStatistics ingress, LongSummaryStatistics egress,
             long connectionDemand, double zonePercentage) {
         // use the override if present
+        int desiredReplicaCount = nodeInformer.getList().size() > 0 ? 1:0;
         if (azReplicaCount.isPresent()) {
             return azReplicaCount.get();
+        } else if (nodeInformer.getList().size() > 3){
```

Comment:
This should be removed - we don't want to force two replicas anymore for each az.

Reply:
removed